core/parsigex/parsigex.go

```diff
@@ -130,3 +145,70 @@ func (m *ParSigEx) Broadcast(ctx context.Context, duty core.Duty, set core.ParSi
 func (m *ParSigEx) Subscribe(fn func(context.Context, core.Duty, core.ParSignedDataSet) error) {
 	m.subs = append(m.subs, fn)
 }
+
+// NewEth2Verifier returns a verifyFunc instance which is responsible to verify the given partial signatures.
```

Comment:
nit: `NewEth2Verifier returns a partial signature verification function for core workflow eth2 signatures`